README.md

```diff
@@ -100,6 +101,13 @@ module.exports = {
 
     - Replace the values of project_id's and suite_id's with the corresponding values specific to your project..
 
+- **`create_missing_cases`** - Default is `true`
```

Comment:
I don't think so, looks default value for this `false`, @gevorg-vardanyan-im could you please recheck?